stl/inc/compare

```diff
@@ -9,7 +9,6 @@
 #include <yvals_core.h>
 #if _STL_COMPILER_PREPROCESSOR
 
-// TODO (for P0768R1 completion):

```

Comment:
As I mentioned a minute ago in my reply to your question in #200, can you keep this comment, but change `TODO` to `TRANSITION` ?